pkg/kvstore/allocator/allocator_test.go

```diff
@@ -100,7 +100,7 @@ func (s *AllocatorSuite) TestSelectID(c *C) {
 
 func (s *AllocatorSuite) BenchmarkAllocate(c *C) {
 	allocatorName := randomTestName()
-	maxID := ID(c.N)
+	maxID := ID(256 + c.N)
```

Comment:
There's a `identity.MinimalNumericIdentity.Uint32()`

Reply:
We can't import it without a massive import cycle refactoring. We have to rethink our packages structure.